recomendar/README.md

```
@@ -0,0 +1,200 @@
+以下の方法で測定データを収集できる。
+
+```bash
+rebar3 shell
+> recomendar:main([]).
+```
+
+処理完了後、`priv/` に測定結果 (`.dat`) が生成される。
+
+### 4-2. 出力ファイルと評価指標
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified
null

(MD040, fenced-code-language)